pipelines/azure-pipelines.yml

```diff
@@ -113,28 +113,6 @@ jobs:
           verbosity: "Verbose"
           alertWarningLevel: "High"
 
-      - task: notice@0
```

Comment:
Normally we would temporarily disable this task for pr validation pipeline by adding a condition on the task. Then when Component Governance team fixed the issue after the follow up, reenable the task.
https://github.com/microsoft/winget-cli/pull/775/files

Reply:
I decided to make the task continue on error. The reason is that this only fails on forked PRs, and would still work ona branch from the source repo. That way we can still update the NOTICE.txt file every so often, and if the CG team is able to fix the issue, it would also work for forked PRs.